### ISSUE TITLE
Does having INFORMATION_SCHEMA and information_schema break dbt docs?

### ISSUE THREAD

**Issue by mharrisb1** [ClickHouse/dbt-clickhouse#74]
Seeing the reason we might possibly need both here https://github.com/ClickHouse/ClickHouse/discussions/33548 but when running `dbt docs generate` I get the following:

```
Compilation Error
  Expected only one database in get_catalog, found [<InformationSchema INFORMATION_SCHEMA>, <InformationSchema INFORMATION_SCHEMA>]
```

Seems similar to this issue  in dbt-mysql https://github.com/dbeatty10/dbt-mysql/issues/58

Output of `dbt debug`

```
17:00:33  Running with dbt=1.1.1
dbt version: 1.1.1
python version: 3.10.4
python path: <REDACTED>
os info: macOS-12.2.1-arm64-arm-64bit
Using profiles.yml file at <REDACTED>
Using dbt_project.yml file at <REDACTED>

Configuration:
  profiles.yml file [OK found and valid]
  dbt_project.yml file [OK found and valid]

Required dependencies:
 - git [OK found]

Connection:
  driver: None
  host: <REDACTED>
  port: 8443
  user: <REDACTED>
  schema: default
  secure: True
  verify: False
  connect_timeout: 10
  send_receive_timeout: 300
  sync_request_timeout: 5
  compress_block_size: 1048576
  compression: 
  custom_settings: None
  Connection test: [OK connection ok]

All checks passed!
```

**Comment by guykoh** [comment]
Hi @mharrisb1, which dbt-clickhouse version are you using? I tried reproducing this issue locally and in front of clickhouse cloud instance, and in both cases I didn't receive any compilation errors.

I see you're using the default database as your schema, this is not recommended with dbt, maybe this could be the issue?

**Comment by guykoh** [comment]
I see now your problem, yes, you can't have 2 schema's with the same name (case insensitive).
You're right that this issue was already reported [here](https://github.com/ClickHouse/ClickHouse/discussions/33548).

Regardless, it is not recommended to use the default database in dbt (see comment in [README](https://github.com/ClickHouse/dbt-clickhouse#database)), so I suggest to use a different schema in your profile and avoid creating duplicated table names

**Comment by mharrisb1** [comment]
Thanks for the reply @guykoh 

I removed the use of the `default` in our project and I still see the same issue. What additional info can I give you to help debug further?

**Comment by guykoh** [comment]
First of all, which clickhouse and dbt-clickhouse versions are you using?
If they are not the latest, is it possible for you to update the versions and check again?

Can you share your profile.yml and your models/*.yml files as well?

**Comment by guykoh** [comment]
I found the reason why you get this error message. You are using different quoting policies in your sources, which causes dbt to store 2 types of information schemas.
Right now, we limit this to 1, although I'm not sure why. I created a [PR](https://github.com/ClickHouse/dbt-clickhouse/pull/82) for removing that limitation, that suppose to resolve your issue (I verified it locally with the lua-dbt project).
